packages/devtools/test/schema.test.ts

```diff
@@ -106,6 +102,36 @@ describe('schema', () => {
     })
   })
 
+  it('@documentAccount is only valid for DIDs', () => {
+    expect(() => {
+      compositeModelsAndCommonEmbedsFromGraphQLSchema(`
+      type ModelWithInvalidDocumentAccountProp @model(
+        accountRelation: LINK,
```

Comment:
`LINK` should be changed to `SINGLE`, I have a branch with this change so let's merge it first and make sure all the changes in this PR match the spec.